### ISSUE TITLE
Increase time for audio recording for whisper

### ISSUE THREAD

**Issue by ManuXD32** [nextcloud/integration_openai#55]
### Describe the feature you'd like to request

Currently we can only record up to 120s to transcribe or translate. Is there a way to increase it up to an hour or customize it in any way?

### Describe the solution you'd like

To be able to record indefinetly.

### Describe alternatives you've considered

None

**Comment by ER-EPR** [comment]
Definitely! 2min is too short, and I think transcribe audio file in nextcloud or through upload is also needed.
